### PR TITLE
HSEARCH-4009 PropertyBridge on property marked with @IndexingDependency(derivedFrom = ...) leads to bootstrap failure

### PR DESCRIPTION
**Pull request by yrodiere**
https://hibernate.atlassian.net/browse/HSEARCH-4009

### CODE REVIEW

File: integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/model/TransientPropertyIT.java [hunk 1]
```diff
@@ -224,5 +283,104 @@ public Integer getAPlusB() {
...
+			}
+
+			@Override
+			public void write(DocumentElement target, Object bridgedElement, PropertyBridgeWriteContext context) {
+				Addition addition = (Addition) bridgedElement;
+				target.addValue( fieldRef, addition.left + addition.right );
+			}
+		}
+	}
+
```
Comment (fax4ever) [inline integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/model/TransientPropertyIT.java:384]: extra space

File: integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/model/TransientPropertyIT.java [hunk 2]
```diff
@@ -106,6 +114,57 @@ public void withDerivedFrom() {
...
+		// C is not used to derive the transient property, so it should not trigger reindexing.
+		OrmUtils.withinTransaction( sessionFactory, session -> {
+			EntityWithDerivedFromAndBridge entity1 = session.load( EntityWithDerivedFromAndBridge.class, 1 );
+			entity1.setC( 42 );
+
+			session.persist( entity1 );
+		} );
+		backendMock.verifyExpectationsMet();
+	}
```
Comment (fax4ever) [inline integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/model/TransientPropertyIT.java:166]: great test!

**Review by fax4ever — APPROVED** [review]
just a trivial comment here, thanks

**Comment by yrodiere** [comment]
Merged, thanks!

**Pull request merged**